course_UvA-DL/activation-functions/.meta.yml

```diff
@@ -3,6 +3,7 @@ author: Phillip Lippe
 created: 2021-08-27
 updated: 2021-08-27
 license: CC BY-SA
+build: 0
```

Comment:
I think none of these builds is needed as tags are not part of notebook rendering